tests/init/config-initializer.js

```diff
@@ -115,8 +115,8 @@ describe("configInitializer", () => {
         npmFetchPeerDependenciesStub.resetHistory();
     });
 
-    after(() => {
-        sh.rm("-r", fixtureDir);
+    after(async () => {
+        await fsp.rm(path.join(fixtureDir, "../../.."), { recursive: true });
```

Comment:
This should also remove "tmp" in the current directory, not just "tmp/eslint/fixtures/config-initializer".